exporter/templates_test.go

```diff
@@ -52,18 +52,9 @@ func xmlCodeRootNode(codeDisplay models.CodeDisplay) *xml.ElementNode {
 }
 
 func TestReasonTemplate(t *testing.T) {
-	// do not negate reason, r2 compatable
+	// do not negate reason
 	reason := models.CodedConcept{Code: "REASON_CODE_1", CodeSystem: "2.16.840.1.113883.6.1"} // specified in cms9_26.json
 	rootNode := xmlReasonRootNode(reason, false, true)
-	assertXPath(t, rootNode, "//entryRelationship", map[string]string{"typeCode": "RSON"}, nil)
```

Comment:
Any reason all the test cases are suddenly gone? I actually like them cut down so you're not checking for redundant stuff, but I was just curious if it was related to the r2 stuff.